test/stubfolderman.cpp

```diff
@@ -1,7 +1,9 @@
 // stub to prevent linker error
 #include "accountmanager.h"
 
-OCC::AccountManager *OCC::AccountManager::instance() { return static_cast<AccountManager *>(new QObject); }
+static QObject dummy;
+
+OCC::AccountManager *OCC::AccountManager::instance() { return static_cast<AccountManager *>(&dummy); }
```

Comment:
Since it's for a singleton please use Q_GLOBAL_STATIC for the AccountManager instance instead (and that dummy static can then be removed).